README.md

```diff
@@ -1,7 +1,95 @@
-# safeposix-rustc [![Build Status](https://github.com/Lind-Project/safeposix-rust/actions/workflows/lind-selfhost.yml/badge.svg?branch=develop)](https://github.com/Lind-Project/safeposix-rust/actions/workflows/lind-selfhost.yml)
-Rust implementation of SafePOSIX
+# RustPOSIX [![Build Status](https://github.com/Lind-Project/RustPOSIX-rust/actions/workflows/lind-selfhost.yml/badge.svg?branch=develop)](https://github.com/Lind-Project/RustPOSIX-rust/actions/workflows/lind-selfhost.yml)
 
-More implementation details could be found at [wiki](https://github.com/Lind-Project/safeposix-rust/wiki).
+# Table of Contents
 
-## Run SafePOSIX-Rust
-See reference at [Run RustPOSIX Independently][(https://github.com/Lind-Project/safeposix-rust/wiki/Run-RustPOSIX-Independently](https://github.com/Lind-Project/safeposix-rust/wiki/Run-SafePOSIX%E2%80%90rust-(RustPOSIX)-Independently))
+- [RustPOSIX ](#rustposix-)
+- [Table of Contents](#table-of-contents)
+  - [Development Guideline](#development-guideline)
+  - [Run RustPOSIX-Rust](#run-rustposix-rust)
+  - [Test RustPOSIX-Rust](#test-rustposix-rust)
+  - [Debugging](#debugging)
+  - [FAQs](#faqs)
+
+RustPOSIX refers to a library operating system (libOS) built using the Rust programming language as part of the Lind project.
+It is a component of the Lind sandbox that provides a subset of the POSIX API (for things like networking, file I/O, etc.) to applications running within the Google Native Client (NaCl) sandbox. leveraging Rust's memory safety and concurrency safety guarantees to provide a secure implementation of OS functionality.
+
+More implementation details could be found at [wiki](https://github.com/Lind-Project/RustPOSIX-rust/wiki).
+
+![alt text](docs/RustPOSIX-README.jpg)
+
+## Development Guideline
+
+- All PRs should be merged to the Develop branch
+
+- Any imports from the standard library or any crates should be done in an interface file
+
+More detailed guideline will be in [RustPOSIX's wiki](https://github.com/Lind-Project/RustPOSIX-rust/wiki/Style-Guide)
+
+## Run RustPOSIX-Rust
+
+Quick start
+Use Develop branch for the most stable behaviour.
+
+```
+docker build -t --platform <your platform> <image_name> .devcontainer
+docker run -it <image_name>
+
+```
+
+This will create a quick container with rustposix build at your local changes.
+helpful for exploration and easy testing.
+
+See reference at [Run RustPOSIX Independently](https://github.com/Lind-Project/RustPOSIX-rust/wiki/Run-Independently)
+
+## Test RustPOSIX-Rust
+
+Use Develop branch for the most stable behaviour.
```

Comment:
I think "up to date" behavior, main should be most stable

Reply:
updating